sdk/go/common/apitype/stacks.go

```diff
@@ -83,6 +83,16 @@ type Log3rdPartyDecryptionEvent struct {
 	CommandName string `json:"commandName,omitempty"`
 }
 
+// BulkDecryptValueRequest defines the request body for bulk decrypting secret values.
+type BulkDecryptValueRequest struct {
+	Secrets [][]byte `json:"secrets"`
+}
+
+// BulkDecryptValueResponse defines the response body for bulk decrypted secret values.
+type BulkDecryptValueResponse struct {
+	DecryptedSecrets map[string][]byte `json:"decryptedSecrets"`
```

Comment:
Not sure I know all this well enough to +1 but seems odd to me that we pass an array in and get a map out? Also everywhere else in thus file refers to Ciphertext and Plaintext, maybe this should match?

Reply:
so the input is a list of Ciphertext in []byte and the response is a map of the ciphertext and decrypted secret

Reply:
Should it be `Ciphertexts` and `Plaintexts` then?

What is the `string` value in the map if all others are `[]byte`? Is there some implied conversion?

Reply:
Folks, we are getting blocked on a naming here....

there is a conversion inside the service that sends the representation back.

Reply:
@Frassle Were you thinking something like this instead?

```
type BulkDecryptValueRequest struct {
	Ciphertexts [][]byte `json:"ciphertexts"`
}
type BulkDecryptValueResponse struct {
	Plaintexts [][]byte `json:"plaintexts"
}
```

Reply:
That's what I would expect given the rest of this file. Same names as elsewhere, no questions of how byte[] is turned into string.